actions/port-forward/action.yaml

```diff
@@ -1,18 +1,22 @@
 name: Port forward Kubernetes service
 description: Creates a port forward for a Kubernetes service and keep it open in the background
 inputs:
-  service:
-    description: Name of Kubernetes service
+  kubectl-version:
+    description: kubectl version, e.g. `v1.23.0` or `latest`
+    default: latest
+    required: false
+  namespace:
+    description: Kubernetes namespace
     required: true
   port:
     description: Port to be forwarded
     required: true
-  namespace:
-    description: Kubernetes namespace
+  service:
+    description: Name of Kubernetes service
     required: true
-  kubectl-version:
-    description: kubectl version, e.g. `v1.23.0` or `latest`
-    default: latest
+  timeout:
+    description: Timeout for portfowrd
+    default: "30"
     required: false
```

Comment:
Why did you change the order, wasn't it logical before? At least required inputs should be first